api/src/test/java/org/apache/iceberg/TestHelpers.java

```diff
@@ -346,6 +350,11 @@ public Long deletedRowsCount() {
       return partitions;
     }
 
+    @Override
+    public ByteBuffer keyMetadata() {
```

Comment:
Is this used in tests? We could just allow this to use the default implementation, unless we added tests that exercise it.